### DISCUSSION TITLE
Plausible in v3 but not in v4?

### DISCUSSION THREAD

**Discussion by DiiiaZoTe** [coollabsio/coolify#1486] [Service Template Requests]
Plausible used to be available in v3. Would it be possible to integrate it back into v4?

**Comment by andrasbacsai** [comment]
We had a trademark issue with them, so I had to remove it.

But you can simply host a Plausible instance by copy-pasting [this file](https://github.com/coollabsio/coolify/blob/main/templates/compose/plausible.yaml) to a docker compose based deployment.
